### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by SamVerschueren**
Did some tweaks because tests are failing.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,15 +31,14 @@ module.exports = str => {
 		throw new TypeError('Expected a string');
 	}
 
-	// used to see if tabs or spaces are the most used
+	// Used to see if tabs or spaces are the most used
```
Comment (julien-f) [inline index.js]: Why these changes?

Please respect the style of existing code :)

Comment (sindresorhus) [reply index.js]: It's what XO enforce now.

Comment (julien-f) [reply index.js]: Ok :)
